language/de-DE/de-DE.ini

```diff
@@ -53,11 +53,19 @@ JALL_LANGUAGE="Alle"
 JAPPLY="Speichern"
 JARCHIVED="Archiviert"
 JASSOCIATIONS="Auch verfügbar:"
+JASSOCIATIONS_ASC="Associations aufsteigend"
+JASSOCIATIONS_DESC="Associations absteigend"
```

Comment:
@joomlagerman/reviewer Wie können wir Associations hier in einen vernünftigen Kontext übersetzen?

Reply:
Sprachverknüpfungen?